Hiba-WK2-HTML_CSS

Hiba-moh opened the pull request.

Hiba-moh committed:
Update index.html

Hiba-moh committed:
creating css and html files

Hiba-moh committed:
css changes

added the Nav styles

Hiba-moh committed:
Revert "css changes"

This reverts commit baa8d909ac8ab8b0db9c586d48d71fc6833ebbac.

Hiba-moh committed:
Header navigater

completed the header navigator

Hiba-moh committed:
backg-image

trying to text over image

Hiba-moh committed:
update

update

Hiba-moh committed:
update css

made the header and the button

Hiba-moh committed:
fonts changed

I've changed fonts and added new section in html file

Hiba-moh committed:
The third section

started to built the third section of the website

Hiba-moh committed:
footer

started to work on the footer

Hiba-moh commented:
Its not my final project so had to close it

Hiba-moh committed:
css changes

styled the content section
styled the photos
changed the font weight for the nav and the content area
changed the button size
changed the background photo size

Hiba-moh committed:
footer changes

inserting the social media icons

Hiba-moh committed:
Trying responsive

first try to make the page responsive ... used relative font size and percentage for margins and sizes

Hiba-moh committed:
Responsive

I made changes to make the webpage responsive

Hiba-moh committed:
css changes

trying to make it responsive

Hiba-moh committed:
Update style.css

Hiba-moh committed:
responsive menu

changes to make the navigation bar responsive

Hiba-moh committed:
nav-bar changes

changed the navigation background color

Hiba-moh commented:
Opened the branch to submit may final work

Rody-Kirwan commented on week2/2-website/css/style.css:
```diff
@@ -15,5 +26,257 @@ body {
+}
+/* ********************************   The links style **************************************/
+.menu li {
+  font-size: 1.3vw;
```
never use `vw` as a unit for font-size or borders.
This is based on the viewport size so it will shrink to nothing very quickly as we get smaller.
Better to use `rem`

Hiba-moh replied:
> never use `vw` as a unit for font-size or borders.
> This is based on the viewport size so it will shrink to nothing very quickly as we get smaller.
> Better to use `rem`

Done

Rody-Kirwan commented on week2/2-website/index.html:
```diff
@@ -1,21 +1,107 @@
+        <h5>Join us on</h5>
+        <ul class="social-media">
+          <li>
+            <img src="img/twitter.svg" alt="twitter icon" />
```
I can't find any images for these. Better to use fontawsome icons
```
<i class="fa fa-twitter"></i>
```

Hiba-moh replied:
> I can't find any images for these. Better to use fontawsome icons
> 
> ```
> <i class="fa fa-twitter"></i>
> ```

Done

Rody-Kirwan commented on week2/2-website/index.html:
```diff
@@ -1,21 +1,107 @@
-    <link href='//fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
-    <link href="//maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css" rel="stylesheet">
-    <link rel="stylesheet" href="css/normalize.css">
+    <link
```
looks like you've added this twice. :)
It won't cause you any problems really but you are making 2 http requests instead of one.
Down the line when we're building apps this is not good for performance

Hiba-moh replied:
> looks like you've added this twice. :)
> It won't cause you any problems really but you are making 2 http requests instead of one.
> Down the line when we're building apps this is not good for performance

There is no duplicate here Rodi  ...

Rody-Kirwan commented on week2/2-website/index.html:
```diff
@@ -1,21 +1,107 @@
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
+  </head>
 
+  <body>
```
You can remove these commented lines. 
In general it is good to remove any comments that are no longer required OR no longer make sense for your work

Hiba-moh replied:
> You can remove these commented lines.
> In general it is good to remove any comments that are no longer required OR no longer make sense for your work

Done

Rody-Kirwan commented on week2/2-website/index.html:
```diff
@@ -1,21 +1,107 @@
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
     <!-- All the images you need are in the 'img' folder -->
+<div class="all">
+    <div class="nav">
```
I like what you are attempting here. 
But if it's not working yet - then it's best to move it to a new branch - as it can be confusing for the person reading your code.
Eitherway - nice attempt :)

Hiba-moh replied:
> I like what you are attempting here.
> But if it's not working yet - then it's best to move it to a new branch - as it can be confusing for the person reading your code.
> Eitherway - nice attempt :)

Thanks ... I removed them

Rody-Kirwan commented on week2/2-website/css/style.css:
```diff
@@ -1,9 +1,20 @@
+  padding: 0;
+  background-color: white;
+  font-family: roboto, sans-serif;
+  font-weight: 400;
```
be careful about setting too much styling on the body element.
I would remove `font-size` `line-height` and `font-weight` as these should be set at a lower level.
Also - no need to set the height - let your content dictate the height :)

Hiba-moh replied:
> be careful about setting too much styling on the body element.
> I would remove `font-size` `line-height` and `font-weight` as these should be set at a lower level.
> Also - no need to set the height - let your content dictate the height :)

Done
`body {
  margin: 0;
  padding: 0;
  font-family: roboto, sans-serif;
  min-height: 100vh;
}
`

Rody-Kirwan commented on week2/2-website/css/style.css:
```diff
@@ -15,5 +26,257 @@ body {
+  margin-bottom: 10rem;
+}
+header {
+  background-color: yellowgreen;
```
I don't think this is needed

Hiba-moh replied:
> I don't think this is needed
 I wanted to make a hamburger menu for the next break point ... I changed my mind and left those lines behind 🙈
 I took them out now ...

Rody-Kirwan commented on week2/2-website/css/style.css:
```diff
@@ -15,5 +26,257 @@ body {
+  min-height: 71px;
+}
+/****************************** The container of the list syle************************* */
+.menu-container {
```
Rather than setting a percentage width on these - It might be better to use a fixed width.
That way it won't bunch up sooner than it should.
Try `width: 675px;`
Then on `.menu` set `justifiy-content: flex-end` and give each `.menu li` `margin: 0 20px`

Hiba-moh replied:
> Rather than setting a percentage width on these - It might be better to use a fixed width.
> That way it won't bunch up sooner than it should.
> Try `width: 675px;`
> Then on `.menu` set `justifiy-content: flex-end` and give each `.menu li` `margin: 0 20px`

It didn't work  :(

Rody-Kirwan commented on week2/2-website/css/style.css:
```diff
@@ -15,5 +26,257 @@ body {
+.first-content button {
+  font-family: Arial, Helvetica, sans-serif;
+  font-weight: 400;
+  font-size: 1.386vw;
```
Again - use `rem` for font-size

Hiba-moh replied:
> Again - use `rem` for font-size

Done

Rody-Kirwan reviewed (approved):
Great work Hiba. 
Very tidy and well formatted. 

I've left some notes to improve. Let me know if you have any questions

Hiba-moh committed:
Changed body tag

removed some tags shouldnt be there and unnecessary ones

Hiba-moh committed:
Body style changes

Removed some body styles

Hiba-moh committed:
Removed some styles

removed label,toggle and all classes from CSS .. because they're no need for them any more ,since I didnt made the hamburger menu.

Hiba-moh committed:
VW,icons and comments

changed vw with rem and deleted any unnecessary comments ... added social media using fontawsome and restyled them .

Hiba-moh reviewed (commented):
Changes after review
